src/code/z_player_lib.c

```diff
@@ -1076,132 +1148,93 @@ void Player_DrawGetItem(GlobalContext* globalCtx, Player* this) {
     }
 }
 
-void func_80090A28(Player* this, Vec3f* vecs) {
-    D_8012608C.x = D_80126080.x;
+void Player_GetSwordTip(Player* this, Vec3f* swordTip) {
+    sSwordTipPos2.x = sSwordTipPos1.x;
 
     if (this->unk_845 >= 3) {
         this->unk_845 += 1;
-        D_8012608C.x *= 1.0f + ((9 - this->unk_845) * 0.1f);
+        sSwordTipPos2.x *= 1.0f + ((9 - this->unk_845) * 0.1f);
     }
 
-    D_8012608C.x += 1200.0f;
-    D_80126098.x = D_8012608C.x;
+    sSwordTipPos2.x += 1200.0f;
+    sSwordTipPos3.x = sSwordTipPos2.x;
 
-    Matrix_MultVec3f(&D_80126080, &vecs[0]);
-    Matrix_MultVec3f(&D_8012608C, &vecs[1]);
-    Matrix_MultVec3f(&D_80126098, &vecs[2]);
+    Matrix_MultVec3f(&sSwordTipPos1, &swordTip[0]);
+    Matrix_MultVec3f(&sSwordTipPos2, &swordTip[1]);
+    Matrix_MultVec3f(&sSwordTipPos3, &swordTip[2]);
 }
 
-void func_80090AFC(GlobalContext* globalCtx, Player* this, f32 arg2) {
-    static Vec3f D_801260C8 = { -500.0f, -100.0f, 0.0f };
-    CollisionPoly* sp9C;
+void Player_DrawHookshotPointer(GlobalContext* globalCtx, Player* this, f32 arg2) {
+    static Vec3f posOffset = { -500.0f, -100.0f, 0.0f };
```

Comment:
the most common term to refer to this is a "reticle", and should be used imo. Also will remove any confusion a programmer may have with pointer lol
I think the DL name should be changed to that as well